pom.xml

```diff
@@ -20,18 +20,18 @@
 
   <groupId>co.cask</groupId>
   <artifactId>trash-plugin</artifactId>
-  <version>1.1.0-SNAPSHOT</version>
+  <version>1.1.1</version>
```

Comment:
develop should be snapshot?

Reply:
Good point - this should be merged against release branch, will change it.